src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Injector;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    private static final Injector daoInjector = Injector.getInstance("mate.academy.dao");
+
+    @Inject
+    private CinemaHallDao cinemaHallDao =
+            (CinemaHallDao) daoInjector.getInstance(CinemaHallDao.class);

```

Comment:
i guess you don't need to get instance here, just create variable of cinemaHallDao and mark it with @Inject